framework/src/utils/ImageSampler.C

```diff
@@ -26,8 +26,8 @@ InputParameters validParams<ImageSampler>()
 
   params.addParam<Point>("origin", "Origin of the image (defaults to mesh origin)");
   params.addParam<Point>("dimensions", "x,y,z dimensions of the image (defaults to mesh dimensions)");
-  params.addParam<unsigned int>("component", "The image component to return, leaving this blank will result in a greyscale value "
-                                             "for the image to be created. The component number is zero based, i.e. 0 returns the first component of the image");
+  params.addParam<unsigned int>("component", "The image RGB-component to return, leaving this blank will result in a greyscale value "
```

Comment:
Yeah, nevermind. The number of components is determined for each image individually with `_data->GetNumberOfScalarComponents();` so an enum wouldn't make sense here (but neither does being overly specific in the doco string, as you cannot guarantee what the components in the image really are).